parlai/core/params.py

```diff
@@ -1071,8 +1071,16 @@ def _handle_hidden_args(self, kwargs):
                 kwargs['help'] = argparse.SUPPRESS
         return kwargs
 
+    def add_recommended_to_help(self, kwargs):
+        """Add recommended value to help string if recommended exists"""
+        if 'recommended' in kwargs:
+            kwargs['help'] += " Recommended: " + kwargs['recommended']
```

Comment:
Make sure you handle the case of help doesn’t exist. I would also prefer recommended to be able to take any type so do a format here please

Reply:
I had actually added that but I assumed that every arg would have a help kw and removed that just before committing, I'll re-add it!
For the formatting, yes good point, will casting as a str suffice ?